crates/subspace-runtime/src/lib.rs

```diff
@@ -481,13 +485,38 @@ impl pallet_transaction_fees::Config for Runtime {
     type WeightInfo = ();
 }
 
+parameter_types! {
+    /// The portion of the `NORMAL_DISPATCH_RATIO` that we adjust the fees with. Blocks filled less
+    /// than this will decrease the weight and more will increase.
+    pub const TargetBlockFullness: Perquintill = Perquintill::from_percent(25);
+    /// The adjustment variable of the runtime. Higher values will cause `TargetBlockFullness` to
+    /// change the fees more rapidly.
+    pub AdjustmentVariable: Multiplier = Multiplier::saturating_from_rational(75, 1_000_000);
+    /// Minimum amount of the multiplier. This value cannot be too low. A test case should ensure
+    /// that combined with `AdjustmentVariable`, we can recover from the minimum.
+    /// See `multiplier_can_grow_from_zero`.
```

Comment:
The test is not present in our codebase and I'm not sure what all these numbers really mean, I assume they are just taken from Polkadot as is.